source/helpers/DateHelpers.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Returns a string with the formatted updatedAt date, in format dd/mm-yyyy
+ * @param {obj} caseObj
+ * */
+export const formatUpdatedAt = updatedAt => {
```

Comment:
Probably better to rename this function to something more generic, since now it really has nothing to do with specifically updatedAt anymore. It really just formats a unix time format to a readable string.

Reply:
Yeah i agree. My imagination on function names was low at the point, so moved on instead of getting stuck on finding a suitable name :D